### PR TITLE
Add Harrison Shoebridge to team page

### PR DESCRIPTION
**Pull request by paked**
Writing about myself isn't really my best skill... I'd appreciate a review and some feedback if it isn't good enough :)

@zachlatta let me know if the photo is too derpy.

### CODE REVIEW

**Comment by paked** [comment]
(also: I can't assign anyone on this PR)

File: app/views/pages/team.html.haml [hunk 1]
```diff
@@ -17,6 +17,16 @@
               universities to bring hackathons to colleges all around the globe.
               Dave enjoys studying hybrid online and offline communities.
           .small-12.medium-6.columns.profile
+            = image_tag 'team/harrison_shoebridge.jpg', class: 'profile-img'
+            .heading
+              %h1 Harrison Shoebridge
+              %h2 Intern
+            %p.bio
+              Harrison joined Hack Club officially in December 2015, previously
```
Comment (MaxWofford) [inline app/views/pages/team.html.haml]: You can drop the date

Comment (paked) [inline app/views/pages/team.html.haml]: @MaxWofford The date occurs in Dave's, Jonathan and Zachs as well. Should we change that too?

**Comment by paked** [comment]
@MaxWofford fixed.

**Comment by zachlatta** [comment]
:heart: 

Thanks :-) Great to finally have you on the page.

**Pull request merged**